docs-ref-conceptual/format-output-azure-cli.md

```diff
@@ -70,7 +75,7 @@ The `yaml` format prints output as [YAML](http://yaml.org/), a plain-text data s
 that format. Some applications and CLI commands take YAML as configuration input, instead of JSON.
```

Comment:
It seems the `...` at L66 breaks GitHub's syntax highlighting:

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/4003950/c8f869e3-ef0a-4c0d-908b-b4a2ae16af72)